assessmentModel/src/androidLibMain/kotlin/android.kt

```diff
@@ -32,6 +29,8 @@ actual object UUIDGenerator {
 }
 
 actual object DateGenerator {
-    actual fun nowString(): String = LocalDateTime.now().format(DateTimeFormatter.ISO_DATE_TIME)
-    actual fun currentYear(): Int = LocalDate.now().year
+    actual fun nowString(): String {
+        return ZonedDateTime.now().format(DateTimeFormatter.ISO_DATE_TIME)
+    }
+    actual fun currentYear(): Int = ZonedDateTime.now().year
```

Comment:
Complete and total nit. Why the different syntax styles? (= vs. {})

Reply:
Uhh... good question.